pkg/platform/packet/packet.go

```diff
@@ -288,6 +294,46 @@ func createTerraformConfigFile(cfg *config, terraformPath string) error {
 	return nil
 }
 
+func validateNodePrivateCIDRs(cfg *config) error {
+	// If both the deprecated and the new fields are provided then error out.
+	if cfg.NodePrivateCIDR != "" && len(cfg.NodePrivateCIDRs) > 0 {
+		//nolint:stylecheck
+		return fmt.Errorf("`node_private_cidr` and `node_private_cidrs` cannot be used together." +
```

Comment:
nit: I'd change this dot to coma, it's a single error message, so we would keep it as a single sentence.